src/components/GameCards/GameCardDefault.vue

```diff
@@ -5,7 +5,7 @@
         <div class="game-info">
             <a v-text="game.name" @click="openDetails" v-if="list.view !== 'covers'" />
 
-            <i class="fas fa-grip-vertical game-drag-handle" />
+            <i v-bind:class="{ 'fas fa-grip-vertical game-drag-handle' : list.sortOrder == 'sortByCustom' || list.sortOrder == null }" />
```

Comment:
same here, you can put the computed prop in `GameCard.js` so it's shared across all gamecard components, hope that makes sense